articles/flutter-vs-reactnative/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/flutter-vs-reactnative/
+title: Flutter vs React Native - Which is a better cross platform framework
+description:
+author:
+date: 2020-10-18T00:00:00-14:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/flutter-vs-reactnative/hero.jpg
+    alt: 
+---
+Today, companies are not just making products, but ecosystems. A gamut of services which can prove to be a steady revenue stream. So it is only natural for them to try to get apps to run across devices in the ecosystem. How about being able to do that with a single codebase? Wishful thinking? That may have been true 20 years ago, but not today.
+<!--more-->
+#### Take 1: One bytecode to rule ‘em all
+The dream of a cross-platform was first braved by Java. It devised the paradigm of [Write Once Run Anywhere (WORA)](https://en.wikipedia.org/wiki/Write_once,_run_anywhere). It was successful to a major extent, according to the Java installer at least.
+
+![Java Installer](/engineering-education/flutter-vs-reactnative/installer.jpeg)
```

Comment:
Lets make sure we are including the source for this image

Reply:
Source for the installer image? Where should I mention it

Reply:
I got what the issue was, the image was not showing up, I have fixed it. Problem with the relative path!

Reply:
I was asking if the image needed a source? 
Where did you get the image file from?